cli.js

```diff
@@ -22,7 +23,9 @@ Usage:
 Options:
 	--length-prefixed  -l  Read input as length-prefixed.
 	                       See https://www.npmjs.com/package/length-prefixed-stream
-	--json  -j             Output JSON instead of a pretty represenation.
+	--json  -j             Output newline-delimeted JSON instead of a pretty represenation.
+												 See http://ndjson.org/
+	--single-json -s			 Outputs standard JSON.
```

Comment:
The spacing should use actual spaces instead of tabs here, as it shouldn't follow the user's choice of tab width.

Reply:
not sure what's wrong with my vscode setup - it should've detected correct settings, let me know if now is better